src/tool-core/models/tool-core/architecture/interfaces/Message/Message.xtuml

```diff
@@ -246,6 +255,17 @@ INSERT INTO EP_PKG_PROXY
 	"00000000-0000-0000-0000-000000000000",
 	"9092a4a9-91f5-4e59-984b-3ca72917240d",
 	'interfaces',
-	'',
+	'Interfaces support passing instances and sets of instances as parameters on Port Messages.
+This is only practical because components can share component independent definitions of
+classes - and, therefore, sets of class instances - through the Package Reference mechanism.
+
+While shared attribute, ssociation and state model definitions of such classes are immutable, 
+within components the classes can be extended with component-specific associations.
+
+Inter-component messages must use component-independent definitions of classes used as 
```

Comment:
Consider the following rewording of the third paragraph:
Since each component-specific class interface includes methods for all associations, including component-specific ones, in which the class participates, 
each method representing an inter-component message must use component-independent interfaces to declare inst_ref and inst_ref_set parameters.  Therefore, methods realizing port-message activities must also use component-independent interfaces when declaring such parameters which are then down-cast to the receiving-component-specific class or class set required by the body of the port-message activity.